src/networkWrapper/wrappers/wsbrowsers.js

```diff
@@ -234,4 +232,25 @@ function poke (listeners, roomId, payload) {
   }
 }
 
+/**
+ * Called when the connection closes with an error state
+ *
+ * @param {boolean} autoReconnect
+ * @param {number} reconnectionDelay
+ * @param {string|Object} message
+ */
+function onClientError(autoReconnect, reconnectionDelay, message) {
```

Comment:
isn't it possible to factorize a little bit the code btw. wsbrowsers.js and wsnode.js ?
(as far as I can see, only the code of `this.connect()` method has some differences in its syntax)